app/views/admin/protocol_drugs/_form.html.erb

```diff
@@ -1,9 +1,15 @@
 <div>
-    <%= bootstrap_form_with(model: [:admin, @protocol, @protocol_drug], local: true, label_errors: true) do |form| %>
-      <%= form.text_field :name, id: :protocol_drug_name %>
-      <%= form.text_field :dosage, id: :protocol_drug_dosage %>
-      <%= form.text_field :rxnorm_code, id: :protocol_drug_rxnorm_code, label: "RxNorm code" %>
-
-      <%= form.primary %>
-    <% end %>
-</div>
\ No newline at end of file
+  <%= bootstrap_form_with(model: [:admin, @protocol, @protocol_drug], local: true, label_errors: true) do |form| %>
+    <%= form.text_field :name, id: :protocol_drug_name %>
```

Comment:
Shall we make the other design changes to this form while we're here? Namely:
* Name, dosage, and Rxnorm on a single line
* "Type" under that
* "Track" under that